websites/S/Sorryfield/presence.ts

```diff
@@ -55,16 +57,20 @@ presence.on("UpdateData", async () => {
 				?.textContent.split(":");
 			if (duration && playing) {
 				const nowTime =
-					Math.floor(Date.now() / 1000) +
-					(parseInt(duration[0].replace("(-", "")) * 60 +
-						parseInt(duration[1].replace(")", ""))) +
-					1;
-				presenceData.endTimestamp = nowTime;
+						Math.floor(Date.now() / 1000) +
+						(parseInt(duration[0].replace("(-", "")) * 60 +
+							parseInt(duration[1].replace(")", ""))) +
+						1,
+					[startTimestamp, endTimestamp] = presence.getTimestampsfromMedia(
+						document.querySelector("video.back")
+					);
+				presenceData.startTimestamp = startTimestamp;
+				presenceData.endTimestamp = endTimestamp;
```

Comment:
```suggestion
				[presenceData.startTimestamp, presenceData.endTimestamp] =  presence.getTimestampsfromMedia(
						document.querySelector("video.back")
					);
```

Reply:
This suggest make some errors.
`ts(2451)` : `Cannot redeclare block-scoped variable 'presenceData'.`
`ts(2493)` : `Tuple type '[number, number]' of length '2' has no element at index '2'.`

I used Github Copilot to resolve these errors, but it reverses the code as it is now.

Reply:
you’re not compiling using our tool.

Reply:
![image](https://github.com/user-attachments/assets/dc89ef42-2c12-4704-91e2-265719912654)

I used PreMiD's compile tool...

Reply:
I found problem in line 106. I will commit it